### PR TITLE
B2C UI test

### PR DESCRIPTION
**Pull request by JoshLozensky**
Made an automated UI test for the B2C sample 4-2.

### CODE REVIEW

File: UiTests/B2CUiTest/B2CUiTest.cs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            _sampleClientAppPath = Path.Join(_samplePath, TC.s_todoListClientPath);
+            _sampleServiceAppPath = Path.Join(_samplePath, TC.s_todoListServicePath);
+        }
+
+        [Fact]
+        [SupportedOSPlatform("windows")]
+        public async Task B2C_ValidCreds_LoginLogout()
+        {
+            // Web app and api environmental variable setup.
+            Dictionary<string, Process>? processes = null;
```
Comment (jennyf19) [inline UiTests/B2CUiTest/B2CUiTest.cs:52]: check the warning, there is one in the GitHub UI

File: UiTests/B2CUiTest/B2CUiTest.csproj [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <!--<TargetFrameworks Condition="'$(TargetNet9)'== 'True'">$(TargetFrameworks); net9.0</TargetFrameworks>-->
```
Comment (jennyf19) [inline UiTests/B2CUiTest/B2CUiTest.csproj:5]: remove commented code?

**Review by jennyf19 — APPROVED** [review]
:shipit:

File: UiTests/B2CUiTest/B2CUiTest.csproj (hunk 2 above)
Comment (JoshLozensky) [reply UiTests/B2CUiTest/B2CUiTest.csproj]: removed

File: UiTests/B2CUiTest/B2CUiTest.cs (hunk 1 above)
Comment (JoshLozensky) [reply UiTests/B2CUiTest/B2CUiTest.cs:52]: enabled nullable in B2C UI test csproj

**Pull request merged**